tsconfig.json

```diff
@@ -0,0 +1,18 @@
+{
+  "compilerOptions": {
+    "allowJs": true,
```

Comment:
Glad this worked, gives people the option to contribute JS now and make TS out of it later